DocumentDataAPI/DocumentDataAPI/Controllers/BiasSchema/BiasDocumentController.cs

```diff
@@ -0,0 +1,122 @@
+using DocumentDataAPI.Data.Repositories.BiasSchema;
+using DocumentDataAPI.Models.BiasSchema;
+using Microsoft.AspNetCore.Mvc;
+using System.Net.Mime;
+
+namespace DocumentDataAPI.Controllers.BiasSchema;
+
+[ApiController]
+[Route(RoutePrefixHelper.Prefix + "/bias_documents")]
+[Produces(MediaTypeNames.Application.Json)]
+public class BiasDocumentController : ControllerBase
+{
+    private readonly ILogger<BiasDocumentController> _logger;
+    private readonly IBiasDocumentRepository _repository;
+
+    public BiasDocumentController(ILogger<BiasDocumentController> logger, IBiasDocumentRepository repository)
+    {
+        _logger = logger;
+        _repository = repository;
+    }
+
+    /// <summary>
+    /// Adds the documents from the content body to the database and returns a sequential list of IDs for the inserted documents.
+    /// </summary>
+    /// <response code="200">Success: A list of IDs for the added documents (i.e., the last inserted ID is last in the list).</response>
+    /// <response code="500">Internal Server Error: a <see cref="ProblemDetails"/> describing the error.</response>
+    [HttpPost]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<List<long>>> InsertDocuments([FromBody] List<BiasDocumentModel> documents)
+    {
+        try
+        {
+            IEnumerable<long> insertedIds = await _repository.AddBatch(documents);
+            return Ok(insertedIds);
+        }
+        catch (Exception e)
+        {
+            _logger.LogError(e, "Unable to add document.");
+            return Problem(e.Message);
+        }
+    }
+
+    /// <summary>
+    /// Persists the changes to the given <paramref name="biasDocumentModel"/> in the database.
+    /// </summary>
+    /// <response code="200">Success: The updated document.</response>
+    /// <response code="204">No Content: Nothing is returned.</response>
+    /// <response code="500">Internal Server Error: a <see cref="ProblemDetails"/> describing the error.</response>
+    [HttpPut]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<BiasDocumentModel>> UpdateDocument([FromBody] BiasDocumentModel biasDocumentModel)
+    {
+        try
+        {
+            return await _repository.Update(biasDocumentModel) == 1
+                ? Ok(_repository.Get(biasDocumentModel.Id).Result)
```

Comment:
No use `.Result` på async ting
```suggestion
                ? Ok(await _repository.Get(biasDocumentModel.Id))
```